src/Orleans/Configuration/ConfigurationExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Orleans.Runtime.Configuration
+{
+    public static class ConfigurationExtensions
```

Comment:
Why not simply an instance method on `ClusterConfiguration` class instead of the extension class?